File: backend/rpg-manager/src/main/java/com/allan/rpg_manager/domain/Talent.java
package com.allan.rpg_manager.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;

import java.util.Optional;

@Getter
@AllArgsConstructor
public class Talent {
    private String name;
    private String description;
    private int levelRequired;
    private Optional<Skill> requiredSkill;
}

File: frontend/rpg-manager/src/app/models/Talent.ts
export class Talent{
    id:number
    name:string
    description:string
    levelRequired:number
    
    constructor(id:number,name:string,description:string,levelRequired:number){
        this.id = id;
        this.name = name;
        this.description = description;
        this.levelRequired = levelRequired
    }
}